### ISSUE TITLE
Admin can approve or reject a membership request

### ISSUE THREAD

**Issue by LobeTia** [Schroedinger-Hat/peoplez#33]
Admins should be able to see all the membership request and manually approve or reject them.

An approved membership moves to status active
A rejected membership moves to status rejected

When admin wants to reject a membership a modal prompting the reason of the reject should be displayed.
Rejection should be logged and properly communicated to the user
A rejection should also trigger a reimburse for the Stripe invoice that was payed by the user

**Comment by 404answernotfound** [comment]
Does this mean we would cover the % of stripe cost per transaction in the case of rejection?

**Comment by TheJoin95** [comment]
Yes, we are covering the costs (as per any services it's working like that, tbh)
